vispy/gloo/preprocessor.py

```diff
@@ -45,7 +45,7 @@ def replace(match):
                 logger.critical('"%s" not found' % filename)
                 raise RuntimeError("File not found", filename)
             text = '\n// --- start of "%s" ---\n' % filename
-            text += remove_comments(open(path).read())
+            text += open(path).read()
```

Comment:
Thinking back, it's possible I was seeing `ResourceWarnings` from opening other types of objects -- and that might have prompted me to try to use `with` whenever possible.